sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -287,19 +340,22 @@ private RecordedData readDataFromFile() {
         try (BufferedReader reader = Files.newBufferedReader(recordFile.toPath())) {
             return RECORD_MAPPER.readValue(reader, RecordedData.class);
         } catch (IOException ex) {
-            throw logger.logExceptionAsWarning(new UncheckedIOException(ex));
+            throw LOGGER.logExceptionAsWarning(new UncheckedIOException(ex));
         }
     }
 
-    /*
-     * Creates a File which is the session-records folder.
+    /**
+     * Get the {@link File} pointing to the folder where session records live.
+     * @return The session-records folder.
+     * @throws IllegalStateException if the session-records folder cannot be found.
      */
-    private File getRecordFolder() {
+    public static File getRecordFolder() {
```

Comment:
Does this need to be public here or can we move this into an implementation utility class?

Reply:
Moved it to a util class.